seisflows/workflow/forward.py

```diff
@@ -425,10 +425,20 @@ def evaluate_objective_function(self, save_residuals=False, **kwargs):
                          "objective function")
             return
 
+        if save_residuals:
```

Comment:
See comment for `evaluate_initial_misfit`. If you decide that is okay, then we can drop the `it="1"` here